src/components/organisms/Badges/Badges.tsx

```diff
@@ -1,77 +1,108 @@
-import React, { useMemo } from "react";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import _ from "lodash";
```

Comment:
Can we import the explicit bits we need from the direct path, rather than the entire lib.

@thecadams I wonder if we can set an eslint ban on this pattern too? Without affecting the more 'specific' imports.